spec/index.bs

```diff
@@ -13999,6 +13999,18 @@ interface GPUCanvasContext {
                 1. Let |descriptor| be the
                     [$GPUTextureDescriptor for the canvas and configuration$](|this|.{{GPUCanvasContext/canvas}}, |configuration|).
                 1. Set |this|.{{GPUCanvasContext/[[configuration]]}} to |configuration|.
+
```

Comment:
Just realized we should also add a link to the [definition of `toneMapping`](https://gpuweb.github.io/gpuweb/#dom-gpucanvasconfiguration-tonemapping) e.g.:

`Note: If an implementation doesn't support HDR WebGPU canvases, it should also not expose this member, to allow for feature detection. See {{GPUCanvasContext/getConfiguration()}}.`